res/values-pt-rBR/strings.xml

```diff
@@ -1247,7 +1247,7 @@ Esta é uma lista de coisas que são *PROIBIDAS* para usar nas skins personaliza
     <string name="Name_Scale_">"Tamanho do Nome:"</string>
     <string name="poor_connection_quality_warning" translatable="true" tools:ignore="Untranslatable">"Seu dispositivo tem uma conexão ruim com o servidor. Isso pode causar lag. Por favor, verifique o seu serviço de Wi-Fi ou celular ou tente uma conexão de internet diferente. Também verifique se você está jogando no servidor mais próximo de você."</string>
     <string name="Button_3D">Botão 3D</string>
-    <string name="Hide_Button_">Ocultar Botão?</string>
+    <string name="Hide_Button_">Ocultar Botão</string>
```

Comment:
Interrogation should keep based on `res/values` (English file).

Reply:
Pronto, fiz a alteração e coloquei a pontuação novamente conforme pedido